examples/LimitToTest.php

```diff
@@ -35,9 +37,25 @@ public function testTimeIntervalToRunForCanBeConfiguredButItNeedsToProduceAtLeas
 
     public function testTimeIntervalToRunForCanBeConfiguredAndAVeryLowNumberOfIterationsCanBeIgnored()
     {
-        $this->minimumEvaluationRatio(0.0)
-             ->limitTo(new DateInterval("PT2S"))
-             ->forAll(
+        $this
+            ->minimumEvaluationRatio(0)
+            ->limitTo(new DateInterval('PT2S'))
+            ->forAll(
+                Generator\int()
+            )
+            ->then(function ($value) {
+                usleep(100 * 1000);
+                $this->assertTrue(true);
+            });
+    }
+
+    /**
+     * @eris-ratio 0
+     * @eris-duration PT2S
```

Comment:
the usage of the annotations look clear so far. Evaluating separately the implementation